app/uk/gov/hmrc/gatekeeper/models/Application.scala

```diff
@@ -432,7 +425,7 @@ object CollaboratorRole extends Enumeration {
   implicit val format = Json.formatEnum(CollaboratorRole)
 }
 
-case class ApplicationState(name: State = State.TESTING, requestedByEmailAddress: Option[String] = None, verificationCode: Option[String] = None, updatedOn: DateTime = DateTime.now()) {
+case class ApplicationState(name: State = State.TESTING, requestedByEmailAddress: Option[String] = None, verificationCode: Option[String] = None, updatedOn: LocalDateTime = LocalDateTime.now()) {
```

Comment:
Do these not need to be Zoned?
I am not certain myself but I think it's worth asking.

Reply:
Like LocalDateTime.now(ZoneOffset.UTC)

Reply:
I really hope we are not mixing zones in now...